BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m

```diff
@@ -66,7 +65,9 @@ + (UIImage *)box_smallIconForItem:(BOXItem *)item
         NSString *fileExtension = [item.name box_pathExtensionAccountingForZippedPackages].lowercaseString;
         NSString *iconName = [self iconNameForFileExtension:fileExtension];
 
-        icon = [UIImage box_iconWithName:[@"small_" stringByAppendingString:iconName]];
+        if (iconName) {
```

Comment:
:) That can't happen and wouldn't be a crash if we didn't have `nil`, but I guess it wouldn't hurt.